### PR TITLE
inactive cases are hidden in assignment selector

### LINKED ISSUE #1184: deactivated casa_cases should not be available to assign to volunteers
dependent on #1182

AS a **supervisor** or **admin**,
I WANT TO _only_ see active `casa_cases` available to assign to volunteers,
SO THAT I do not accidentally assign a volunteer to an inactive `casa_case`.

### How it works now
Currently, all `casa_cases` display as available to assign to volunteers in the dropdown menu under "Assign a New Case" on the Editing Volunteer Page (see screenshot below).
Screen Shot 2020-10-24 at 5 40 27 PM

### How it should work
Only active `casa_cases` should display in the dropdown menu under "Assign a New Case" as available to assign to a volunteer.

### PR DESCRIPTION
**Pull request by chaserx**
### What github issue is this PR for, if any?
Resolves #1184 

### What changed, and why?

Added a `active` scope on `CasaCase` and chained that scope onto the `not_assigned` scope used in `ui_helper` to display available cases for assignment. 

This is important to prevent the accidental assignment of an inactive case to a volunteer. 

### How will this affect user permissions?
- Volunteer permissions: no affect
- Supervisor permissions: no affect
- Admin permissions: no affect

### How is this tested? (please write tests!) 💖💪

Added a system test and tested manually

### Screenshots please :)

Screen Shot 2020-10-30 at 8 58 52 PM

Screen Shot 2020-10-30 at 9 01 21 PM

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

awesome

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -299,9 +299,6 @@ GEM
     standard (0.8.1)
       rubocop (= 1.0.0)
       rubocop-performance (= 1.8.1)
-    sweetalert2-rails (0.1.1)
-    sweetify (2.0.0)
-      actionpack
```
Comment (chaserx) [inline Gemfile.lock:304]: I suspect that these were not checked in last time. I didn't adjust the Gemfile, but I'm happy to invesigate further if needed.

**Comment by jmkoni** [comment]
You just have a standard failure. Don’t forget to run `standardrb --fix`

**Review by jmkoni — APPROVED** [review]
Simple and good test coverage! Just make sure the build passes and we can merge 😃

**Comment by chaserx** [comment]
> You just have a standard failure. Don’t forget to run `standardrb --fix`

Whoops! Fixed.

**Comment by jmkoni** [comment]
@chaserx  thanks for your great work on this!

**Comment by compwron** [comment]
You are invited to the CASA asynchronous retrospective-and-futurespective! Please take a look at https://bit.ly/casaretro (no login required) and add at least one note! Even if it's just "I was here and you are cool" or "omg y'all why do you keep messaging me" we want to hear it. <3
Next week Wednesday 18 November @ 6pm Pacific time we will Zoom about the board and talk about themes and takeaways. (Zoom link in our calendar at http://bit.ly/casacal or slack #casa channel https://rubyforgood.herokuapp.com/ )
Please ESPECIALLY add lots of notes if you won't be able to make it on Wednesday to talk!
* Note: the board will be monitored for spam; message me or any team lead if you see a problem
https://bit.ly/casaretro

**Pull request merged**